src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/AbstractFlowPass.vb

```diff
@@ -2528,6 +2528,19 @@ lUnsplitAndFinish:
             Visit(node.Expression)
             Return Nothing
         End Function
+
+        Public Overrides Function VisitInterpolatedStringExpression(node As BoundInterpolatedStringExpression) As BoundNode
+            For Each item In node.Contents
+                Visit(item)
+            Next
+
+            Return Nothing
+        End Function
+
+        Public Overrides Function VisitInterpolation(node As BoundInterpolation) As BoundNode
+            Visit(node.Expression)
+            Return Nothing
```

Comment:
At the moment they can't be other than constant expressions, but it couldn't hurt to be explicit here. I'll add them, just in case we relax this restriction in the future.